### ISSUE TITLE
About icdar2015 dataset

### ISSUE THREAD

**Issue by chenf99** [ying09/TextFuseNet.pytorch#76]
Hello，I want to train TextFuseNet on icdar2015 dataset, it seems that I should convert the txt of train/test set to json. However, in `configs/ocr/icdar2015_101_FPN.yaml`, both the  TRAIN dataset and TEST dataset are icdar2015. 
According to `detectron2/data/datasets/builtin.py`, the json of "icdar2015" is icdar2015/train.json. So, is the yaml file wrong？Why the train set and the test set are the same?

**Comment by Real-YeJ** [comment]
@chenf99  When testing, please run the code in demo (icdar2015_detection.py), and the detection results of test images will be saved as txt format. For evaluation, please refer to the link of icdar2015 in Evaluation part.
